cmd/simv3/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/smartcontractkit/ocr2keepers/cmd/simv3/run"
 	"github.com/smartcontractkit/ocr2keepers/cmd/simv3/simulators"
 	"github.com/smartcontractkit/ocr2keepers/cmd/simv3/telemetry"
-	"github.com/smartcontractkit/ocr2keepers/pkg/encoding"
+	"github.com/smartcontractkit/ocr2keepers/pkg/v2/encoding"
```

Comment:
not sure why v2 encoding was referenced in simv3, ignoring it for now

Reply:
The few v2 components in the v3 sim that I saw as part of the type refactoring, weren't actually being used, so I guess it was maybe just a copy-paste artifact?

Reply:
i tried removing them but ran into errors so let them be for now

Reply:
actually was able to remove them now i think :D